From b7c65622eb1dd64a4252e441910809d3c66ed49f Mon Sep 17 00:00:00 2001
Message-Id: <b7c65622eb1dd64a4252e441910809d3c66ed49f.1549534460.git.liu.denton@gmail.com>
In-Reply-To: <cover.1549534460.git.liu.denton@gmail.com>
References: <cover.1549521103.git.liu.denton@gmail.com>
	<cover.1549534460.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Sat, 12 Jan 2019 13:30:44 -0500
Subject: [PATCH v3 1/3] git-submodule.txt: "--branch <branch>" option defaults
 to 'master'
To: git@vger.kernel.org
Cc: gitster@pobox.com

This behavior is mentioned in gitmodules.txt but not in
git-submodule.txt so we copy the information over so that it is not
missed.

Also, add the missed argument to the -b/--branch option.

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---
 Documentation/git-submodule.txt | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/Documentation/git-submodule.txt b/Documentation/git-submodule.txt
index ba3c4df550..4150148fa3 100644
--- a/Documentation/git-submodule.txt
+++ b/Documentation/git-submodule.txt
@@ -255,13 +255,14 @@ OPTIONS
 	This option is only valid for the deinit command. Unregister all
 	submodules in the working tree.
 
--b::
---branch::
+-b <branch>::
+--branch <branch>::
 	Branch of repository to add as submodule.
 	The name of the branch is recorded as `submodule.<name>.branch` in
 	`.gitmodules` for `update --remote`.  A special value of `.` is used to
 	indicate that the name of the branch in the submodule should be the
-	same name as the current branch in the current repository.
+	same name as the current branch in the current repository.  If the
+	option is not specified, it defaults to 'master'.
 
 -f::
 --force::
-- 
2.20.1.522.g5f42c252e9

